docs/getting-started/docker.rst

```diff
@@ -120,4 +120,33 @@ Perform the actual code generation step with ``docker run``:
     beneath this for compilation to succeed.
 
 
+Generating Samples
+~~~~~~~~~~~~~~~~~~
+
+In addition to generating client libraries, the generator can also create standalone
+executable code samples.
+The user can specify individual sample config files or can pass paths to directories
+that contain sample configs. Directories are searched recursively, and any file that
+is not a sample config is ignored.
+
+A full description of the sample config, generated manifest, and generated samples
+is outside the scope of this documentation.
+
+.. code-block:: shell
+
+  # Multiple sample paths or directories can be passed simultaneously by duplicating
+  # the 'samples' option.
+  # If no 'samples' option is passed, the generator does not generate a manifest.
+  $ docker run \
+    --mount type=bind,source=$(pwd)/path/to/proto/dir,destination=/in/path/to/proto,readonly \
+    --mount type=bind,source=$(pwd)/dest/,destination=/out/ \
+    --rm \
+    --user $UID \
+    --samples=path/to/sample/config.yaml \
+    --samples=path/to/sample/dir/ \
```

Comment:
`--samples` should be flags of the actual command instead of `docker run`. When I run this command I got:
```
unknown flag: --samples
See 'docker run --help'.
```

Reply:
I _think_ this is solved by changing '=' to ' ', or at least `./docker-entrypoint.sh --samples /tmp/` gives no errors. Updated in the doc.

Reply:
I think you need to move `--samples` after `gcr.io/gapic-images/gapic-generator-python` to get the flags passed to `docker-entrypoint.sh`. Otherwise they are passed to `docker run`.

Reply:
You're right. A more careful reading of the existing documentation agrees with you. Fixing.